competitions/s1floods/benchmark_tutorial.ipynb

```diff
@@ -0,0 +1,2029 @@
+    "\n",
+    "In this post, we'll cover:\n",
+    "\n",
+    "- [Exploring the data](#section1)\n",
+    "- [Preparing the data](#section2)\n",
+    "- [Explaining the benchmark model](#section3)\n",
+    "- [Fitting the benchmark model](#section4)\n",
+    "- [Generating a submission in the correct format](#section5)\n",
+    "\n",
+    "Let's get started!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If running on the Planetary Computer, uncomment below\n",
+    "# !pip install watermark"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"section1\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Python implementation: CPython\n",
+      "Python version       : 3.8.10\n",
+      "IPython version      : 7.26.0\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "%load_ext watermark\n",
+    "%watermark -v"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "%watermark -i"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explore the Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we'll download the training data to a new subdirectory called `training_data/` in our current working directory. If you are working on the Planetary Computer Hub, consider using [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) to store your data using the West Europe region, which is where your server is running.\n",
+    "\n",
+    "```\n",
+    "$ tree training_data/\n",
+    "training_data/\n",
+    "├── flood-training-metadata.csv\n",
+    "├── train_features\n",
+    "│   ├── awc00_vh.tif\n",
+    "│   ├── awc00_vv.tif\n",
+    "        ...\n",
+    "│   ├── wvy31_vh.tif\n",
+    "│   └── wvy31_vv.tif\n",
+    "└── train_labels\n",
+    "    ├── awc00.tif\n",
+    "    ├── awc01.tif\n",
+    "        ...\n",
+    "    ├── wvy30.tif\n",
+    "    └── wvy31.tif\n",
+    "\n",
+    "2 directories, 1627 files\n",
+    "```\n",
+    "\n",
+    "To better understand what's in our data, we can begin by exploring trends across different locations and flood events. Let's load the metadata and look at what we have."
```

Comment:
Can you merge the next few cells, just so that this is easier to run through in fewer clicks?